runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableHeader.java

```diff
@@ -93,6 +95,22 @@ public void mouseExited(MouseEvent mouseEvent)
 		textLabel.setText(title);
 		textLabel.setFont(FontManager.getRunescapeSmallFont());
 
+		final JMenuItem refresh = new JMenuItem("Refresh worlds");
+		refresh.addActionListener(e ->
+		{
+			if (onRefresh != null)

```

Comment:
it looks like it is never null? or at least it probably should not be

Reply:
It's about trusting whatever developer comes next I guess

Reply:
Add `@Nonnull` annotation then and it can be changed later.